galleries/team/metadata.yml

```diff
@@ -164,3 +164,8 @@ title: Saul Shanabrook
 github: saulshanabrook
 projects: ['Vega', 'Jupyter', 'metadsl']
 ---
+name: filipe_lains.png
+title: Filipe Laíns
+github: FFY00
+projects: ['Arch Linux', 'pypa/build', 'CPython Triage', 'importlib_resources']
```

Comment:
We try to keep this focused on projects that are in scope for Quansight Labs. Of course that boundary is often slightly fuzzy, however I'd say Arch Linux is not in scope. And `CPython Triage` should just be `CPython` I think - you don't have to have commit rights to list a project, regular contributions are enough.

Reply:
That makes sense. I put it because it demonstrates my experience with packaging, but was not sure.

Reply:
Good point about demo'ing experience. When we talk to people in person we can still mention that of course. For this page the main focus though is things we do as part of Labs.